tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/06_displayDiscountedPrice.js

```diff
@@ -0,0 +1,158 @@
+    const pageTitle = await this.pageObjects.catalogPriceRulesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.catalogPriceRulesPage.pageTitle);
+  });
+
+  it('should create new catalog price rule', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'createCatalogPriceRule', baseContext);
+    await this.pageObjects.catalogPriceRulesPage.goToAddNewCatalogPriceRulePage();
+    const pageTitle = await this.pageObjects.addCatalogPriceRulePage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.addCatalogPriceRulePage.pageTitle);
+    const validationMessage = await this.pageObjects.addCatalogPriceRulePage.createEditCatalogPriceRule(priceRuleData);
+    await expect(validationMessage).to.contains(this.pageObjects.catalogPriceRulesPage.successfulCreationMessage);
+  });
+
+  it('should go to \'Shop parameters > Product Settings\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProductSettingsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.shopParametersParentLink,
+      this.pageObjects.boBasePage.productSettingsLink,
+    );
+    const pageTitle = await this.pageObjects.productSettingsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.productSettingsPage.pageTitle);
+  });
+
+  const tests = [
+    {args: {action: 'disable', unitPrice: false, unitDiscount: true}},
+    {args: {action: 'enable', unitPrice: true, unitDiscount: false}},
+  ];
+  tests.forEach((test) => {
+    it(`should ${test.args.action} display discounted price`, async function () {
+      await testContext.addContextItem(
+        this,
+        'testIdentifier',
+        `${test.args.action}DisplayDiscountedPrice`,
+        baseContext,
+      );
+      const result = await this.pageObjects.productSettingsPage.setDisplayDiscountedPriceStatus(
+        test.args.unitPrice,
+      );
+      await expect(result).to.contains(this.pageObjects.productSettingsPage.successfulUpdateMessage);
+    });
+
+    it('should check the existence of the \'Unit discount\' or the \'Unit price\'', async function () {
+      await testContext.addContextItem(
+        this,
+        'testIdentifier',
+        `checkUnitPrice${this.pageObjects.boBasePage.uppercaseFirstCharacter(test.args.action)}`,
+        baseContext,
+      );
+      page = await this.pageObjects.productSettingsPage.viewMyShop();
+      this.pageObjects = await init();
+      await this.pageObjects.homePage.changeLanguage('en');
+      await this.pageObjects.homePage.goToProductPage(1);
+      const isUnitPriceVisible = await this.pageObjects.productPage.isUnitPriceVisible();
+      await expect(isUnitPriceVisible).to.equal(test.args.unitPrice);
+      const isUnitDiscountVisible = await this.pageObjects.productPage.isUnitDiscountVisible();
+      await expect(isUnitDiscountVisible).to.equal(test.args.unitDiscount);
```

Comment:
```suggestion
      const disountValue = this.pageObjects.productPage.getDiscountValue();
      await expect(discountValue).to.equal(test.args.textToCheck);
```

Following the suggestion on FO/product.js file